core/modules/system/css/system.css

```diff
@@ -60,12 +60,6 @@
  *
  * @see collapse.js
  */
-.js fieldset.collapsed {
```

Comment:
This wasn't doing much for the vanilla (no theme at all) and is annoying to override, just removed it.